Makefile

```diff
@@ -2,6 +2,8 @@
 	    localtest localunittest localintegration \
 	    test unittest integration
 
+PREFIX := $(DESTDIR)/usr
```

Comment:
Well I am not crazy about installing it by default in /usr/local. But if others want it there, I can put it there.